Userland/Services/WindowServer/WindowManager.cpp

```diff
@@ -738,6 +742,56 @@ void WindowManager::start_window_resize(Window& window, MouseEvent const& event,
     start_window_resize(window, event.position(), event.button(), resize_direction);
 }
 
+void WindowManager::start_tile_window_animation(Gfx::IntRect const& starting_rect)
+{
+    m_tile_window_overlay_animation = Animation::create();
+    m_tile_window_overlay_animation->set_duration(150);
```

Comment:
(care to extract this number as a constant to a more prominent location?)

Reply:
It looks like all other animation durations are currently hardcoded this way. I agree with this, but let's make this a separate PR and make it configurable rather than hardcoding it.
```
WindowManager.cpp:    m_tile_window_overlay_animation->set_duration(150);
Window.cpp:    m_animation->set_duration(150);
Window.cpp:    m_animation->set_duration(150);
Overlays.cpp:            m_move_into_overlay_rect_animation.animation->set_duration(150);
Compositor.cpp:    m_window_stack_transition_animation->set_duration(250);
```

Reply:
do not want to bait you into that; thought about simple constant ‘somewhereʼ near the animation machinery. yet it seems itʼs not separated yet (just noticed youʼre editing `WindowManager.cpp`!).

all in all—good as is! 🙇🏻‍♂️